extensions/blocks/podcast-player/templates/playlist-track.php

```diff
@@ -30,16 +32,27 @@
 
 <li
 	class="<?php echo esc_attr( trim( $class ) ); ?>"
+	[class]="'<?php echo esc_attr( trim( 'jetpack-podcast-player__track ' . $secondary_colors['class'] ) ); ?> ' + (currentTrack == <?php echo esc_attr( $index ); ?> ? 'is-active' : '')"
 	style="<?php echo esc_attr( $style ); ?>"
+	option="<?php echo esc_attr( $index ); ?>"
+	<?php echo $is_active ? 'selected' : ''; ?>
 >
+	<?php if ( $is_amp ) : ?>
+	<div class="jetpack-podcast-player__track-link">
+	<?php else : ?>
 	<a
-		class="jetpack-podcast-player__track-link"
 		href="<?php echo esc_url( $attachment['link'] ); ?>"
-		role="button"
 		<?php echo $is_active ? 'aria-current="track"' : ''; ?>
+		role="button"
+		class="jetpack-podcast-player__track-link"
 	>
+	<?php endif; ?>
 		<span class="jetpack-podcast-player__track-status-icon"></span>
 		<span class="jetpack-podcast-player__track-title"><?php echo esc_html( $track_title ); ?></span>
 		<time class="jetpack-podcast-player__track-duration"><?php echo esc_html( $track_duration ); ?></time>
+	<?php if ( $is_amp ) : ?>
+	</div>
+	<?php else : ?>
 	</a>
+	<?php endif; ?>
```

Comment:
It's debatable whether it's worth extract this into a separate template for AMP and Standards. I'd say currently leave in a single file, but with a view to extracting later if we need any more AMP specific markup/logic.

Reply:
I was wondering if an additional indentation could help here. Essentially treating `<?php if…` as an indentation bump, which might make the structure more apparent.